### DISCUSSION TITLE
for github

### DISCUSSION THREAD

**Discussion by OYB0** [community/community#79118] [Programming Help]
### Body

Hello, I have a project in Replit. I created a Telegram bot code. How can I hide the code using github? I mean, when someone runs the project, it starts, but there is no code written. I do not understand anything in github. I want help. 

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by varunpats** [comment]
Hey, seems like you mistakenly forgot to add file-changes while committing code. I suggest you to watch few github video tutorials on YT. That will definitely help you to gain required knowledge in little time. Moreover github is used widely, time you spend on mastering this will surely help you in career. Hope this helps.

If this helps, please click on `Mark as Answer` below. Thanks!
